README.md

```diff
@@ -104,19 +104,33 @@ If you have a custom setup file and want to use this library then add the follow
 import 'jest-expect-message';
 ```
 
-### Configure Typescript
+### Configure TypeScript
 
-Add the following entry to your tsconfig to enable Typescript support.
+If you keep the `types` list of the `compilerOptions` explicit, add `jest-expect-message` to it:
 
 ```json
-  "files": ["node_modules/jest-expect-message/types/index.d.ts"],
+{
+  "compilerOptions": {
+    "types": ["jest", "jest-expect-message"]
+  }
+}
+```
+
+Alternatively if you prefer letting TypeScript to discover the `@types/*` packages automatically, declare `jest-expect-message` as an `@types/*` package in `package.json`:
+
+```json
+{
+  "devDependencies": {
+    "@types/jest-expect-message": "npm:jest-expect-message@*",
```

Comment:
The inspiration for this solution was the [Supporting `lib` from `node_modules`](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-beta/#supporting-lib-from-node_modules) section from TypeScript 4.5 release notes. The usage of `npm:` protocol looked smart. I tried out the same approach with `@types/*` package declaration and it worked! Well.. Looks obvious, but that was rather a discovery for me ;D